### ISSUE TITLE
Uruguay, alternative data source

### ISSUE THREAD

**Issue by 3dgiordano** [sociepy/covid19-vaccination-subnational#54]
In the case of Uruguay, you can take the information from the following repository:
https://github.com/3dgiordano/covid-19-uy-vacc-data/blob/main/data/Uruguay.csv

The same repository that is used in OWID for its data.
I added columns to reflect the sub-national data in the last few weeks.

If you want to migrate, the actual column format is total_[sub_id] where [sub_id] is the identification of the ISO format region in lowercase.
Example: total_ar is total_vaccinations of UY-AR

In the next week I hope to resolve the data for people_vaccinated and people_fully_vaccinated.
I will notify you when it is available.

**Comment by lucasrodes** [comment]
Great, thanks for your work and for letting me know. I'll be changing the script soon 😃

**Comment by 3dgiordano** [comment]
Hi @lucasrodes 

The data structure is already handling the concept of vaccinated and fully vaccinated people both at the country level and by region.
people_id and fully_id where id is the id of the iso region, without the country id.

https://github.com/3dgiordano/covid-19-uy-vacc-data/blob/main/data/Uruguay.csv

Currently the data are estimates based on vaccination plans. We are waiting for official data to begin to be published to update with real data.

We have just started to give the second dose, in the course of the week I hope to have the final data and make adjustments to the actual data.

Regards

**Comment by lucasrodes** [comment]
This is great @3dgiordano ! Thanks for letting me know

I have been looking at your data and seems a good option to migrate for now. I will need to spend some time however as the formating is slightly different and will need to do some column/row transformations.

I'll try to do this in the upcoming days.

Regards

**Comment by lucasrodes** [comment]
Hi @3dgiordano,
I am currently a bit busy at work and can't update the [script](https://github.com/sociepy/covid19-vaccination-subnational/blob/main/src/covid_updater/scraping/countries/uruguay.py) to update Uruguay's data.

I might do it soon, but for the time if you want to work on this I am more than happy. Thanks!

**Comment by 3dgiordano** [comment]
Hi @lucasrodes 

Maybe I can work on it over the weekend.

Next week if you have a PR to review it is because I was able to dedicate it.

Regards.

**Comment by 3dgiordano** [comment]
Hi @lucasrodes 

I did not have time to make all the changes.
I had time to generate in your format, the same thing I do for OWID.

https://github.com/3dgiordano/covid-19-uy-vacc-data/blob/main/data/Subnational.csv

Importing the data now into the python would be to import it directly into the DataFrame without requiring extra transformation.

Change your data source to the raw file in csv and use 
https://raw.githubusercontent.com/3dgiordano/covid-19-uy-vacc-data/main/data/Subnational.csv

**Comment by 3dgiordano** [comment]
Hi @lucasrodes

Apologies for closing the issue, it was a mistake.

Now my project generates in the format of your project.
I did not have the time to make the modifications in your python project to change and test the new source.

Surely I already did the most difficult part and I made it much easier for you.

https://raw.githubusercontent.com/3dgiordano/covid-19-uy-vacc-data/main/data/Subnational.csv

Regards
